test/smoke/test/index.js

```diff
@@ -17,7 +17,7 @@ const suite = opts['web'] ? 'Browser Smoke Tests' : 'Smoke Tests';
 
 const options = {
 	color: true,
-	timeout: 300000,
+	timeout: 600000,
```

Comment:
It would be better to only have the tests that need this long of a timeout to set this - generally tests shouldn't ever be taking this long.